src/connectors/patchman/impl.js

```diff
@@ -32,6 +32,25 @@ module.exports = new class extends Connector {
         ).then(res => _.get(res, ['data'], null));
     }
 
+    getPackage (id, refresh = false) {
```

Comment:
Because of the new endpoint would you be able to add some unit-tests in the src/connectors/patchman/impl.unit.js file?